### DISCUSSION TITLE
Earphones are back

### DISCUSSION THREAD

**Discussion by InclusionProgress** [m-r-s/hearingaid-prototype#33] [General]
The Roland CS-10EM Binaural Microphones/Earphones are available again in Europe. On the Wiki-Side Information are given that this binaural Earphones are not reachable.

**Comment by m-r-s** [comment]
My latest information was that there is still stock but that the hardware is not produced anymore.
But I would be happy to learn that this is not the case.

Also, if there are any alternatives to this ear-level hardware, I would also be happy to hear about it.

**Comment by InclusionProgress** [comment]
Two month before they were out. But onne seller is:
https://www.musikhaus-korn.de/de/suche?q=binaural
But there is company from Germany who produce it for
some more money. A friend of mine said that this
products are much netter. Try: http://www.ohrwurmaudio.eu/index.html
All the best for this year. Happy New Year.

Am So., 3. Jan. 2021 um 09:23 Uhr schrieb Marc René Schädler <
notifications@github.com>:
